spec/controllers/catalog_controller_spec.rb

```diff
@@ -20,62 +20,6 @@
     end
   end
 
-  describe '#show' do
```

Comment:
I removed this whole test because it's duplicated by the `spec/feature/item_view_metadata_spec.rb`